data_rentgen/consumer/extractors/dataset.py

```diff
@@ -76,13 +76,13 @@ def extract_dataset_and_symlinks(dataset: OpenLineageDataset) -> tuple[DatasetDT
     if table_symlinks:
         # We are swapping the dataset with its TABLE symlink to create a cleaner lineage.
         # For example, by replacing an HDFS file with its corresponding Hive table.
-        # This ensures that all operations interact with a single table instead of multiple files (which may represent different partitions).
+        # This ensures that all operations interact with a single table instead of multiple files (which may represent different partitions).  # noqa: E501
         # Discussion on this issue: https://github.com/OpenLineage/OpenLineage/issues/2718
 
         # TODO: add support for multiple TABLE symlinks
         if len(table_symlinks) > 1:
             logger.warning(
-                "Dataset has more than one TABLE symlink. Only the first one will be used for replacement. Symlink name: %s",
+                "Dataset has more than one TABLE symlink. Only the first one will be used for replacement. Symlink name: %s",  # noqa: E501
```

Comment:
Have you set proper line length in pyproject.toml?

Reply:
120.
Formatter can't resolve this situation. And for me also it is strange to have multyline string as warning message